packages/www/docs/guides/start-live-streaming/api-key.mdx

```diff
@@ -11,18 +11,42 @@ You need an API key to make requests of the Livepeer.com API. You do not need to
 create an API key if you are creating and configuring live streams using only
 the Livepeer.com Dashboard.
 
+When you create a new API key, copy and secure the value displayed to you. This
+is the sole secret you will have to use when making requests to the Livepeer.com
+API, so share and store it with caution.
+
 Login to the Livepeer.com Dashboard, and navigate to
 [the API keys page within the Developers tab](https://livepeer.com/dashboard/developers/api-keys).
 On this page, you can manage your API keys including creating or deleting them
 and monitoring their usage.
 
-When you create a new API key, copy and secure the value displayed to you. This
-is the sole secret you will have to use when making requests to the Livepeer.com
-API, so share and store it with caution. We currently allow you to reveal the
-secret of any existing API key but this behavior may change in the future, so
+## How to get an API Key using the Livepeer.com Dashboard
+
+Make sure you register Livepeer Video Service Account and get API Key
+
+- Create a free [Livepeer Video Service Account](https://livepeer.com/register)
+
+![Api Guide 1](/img/docs/api-guide1.png)
+
+- Click `Create API Key`
+
+![Api Guide 1](/img/docs/api-guide2.png)
+
+- A popup should show up. Enter the name for your API key and then Click `Create`.
+
+![Api Guide 3](/img/docs/api-guide3.png)
+
+- Click on `Create Stream` and your dashboard should look like:
+
+![Api Guide 4](/img/docs/api-guide4.png)
```

Comment:
This files should probably have a more specific name. We have plenty of API guides so it can get confusing if we have a bunch of these with no reference to which guide they are used from.

I would go with sth like `/img/docs/api-key-1/2/3/4.png`, with the same name as the `.mdx`

Reply:
+1

Reply:
fixed the folder structure for screen shot